src/LowercaseSentenceTokenizer.java

```diff
@@ -29,8 +30,21 @@ public class LowercaseSentenceTokenizer implements Tokenizer {
    * @return a list of tokens, where each token is a word or a period
    */
   public List<String> tokenize(Scanner scanner) {
+    List<String> wordList = new ArrayList<String>();
     // TODO: Implement this function to convert the scanner's input to a list of words and periods
-    return null;
+    while(scanner.hasNext()) {
+      String item = scanner.next();
+      switch(item.charAt(item.length()-1)) {
+        case '.':
+          wordList.add(item.substring(0,item.length()-1).toLowerCase());
+          wordList.add(item.substring(item.length()-1).toLowerCase());
+          break;
+        default:
+          wordList.add(item.toLowerCase());
+          break;
+      }
```

Comment:
Nice logic! A switch case might be a bit overkill here though. We generally prefer if/else if there are only two cases.